modules/nextflow/src/main/groovy/nextflow/Const.groovy

```diff
@@ -58,12 +58,12 @@ class Const {
     /**
      * The app build time as linux/unix timestamp
      */
-    static public final long APP_TIMESTAMP = 1604244520074
+    static public final long APP_TIMESTAMP = 1610593728268
 
     /**
      * The app build number
      */
-    static public final int APP_BUILDNUM = 5431
+    static public final int APP_BUILDNUM = 5433
 
```

Comment:
Changes to this file should not be committed since these are automatically generated